hotel/src/main/java/com/andersenhotels/presenter/Presenter.java

```diff
@@ -81,28 +105,32 @@ public boolean saveState() {
 
     public boolean loadState() {
         try {
-            this.hotelService = StateManager.loadState();
+            this.hotelService = currentStorage.loadState();
             return true;
         } catch (IOException e) {
             return false;
         }
     }
 
-    public boolean saveStateForTests(String testPath) {
+    public boolean saveStateForTests() {
         try {
-            StateManager.saveStateForTests(hotelService, testPath);
+            currentStorage.saveStateForTests(hotelService);
             return true;
         } catch (IOException e) {
             return false;
         }
     }
 
-    public boolean loadStateForTests(String testPath) {
+    public boolean loadStateForTests() {
         try {
-            this.hotelService = StateManager.loadStateForTests(testPath);
+            this.hotelService = currentStorage.loadStateForTests();
             return true;
         } catch (IOException e) {
             return false;
         }
     }
+
+    public static String getTEST_PATH() {
+        return JsonStorage.getTEST_PATH();
+    }
```

Comment:
why we need this wrap method? can you use just JsonStorage.getTEST_PATH() instead of Presenter.JsonStorage.getTEST_PATH() ?